File: java/PML/src/tags/TagParagraph.java
package tags;

import java.util.ArrayList;

import parser.PMLParser;

public class TagParagraph extends Tag{

	@Override
	public void output(ArrayList<Tag> tags) {
		System.out.println("<p>");
		PMLParser.parse(tags);
		System.out.println("</p>");
	}
}

File: php/classes/PML/Tags/TagParagraph.php
<?php namespace PML\Tags;

class TagParagraph extends Tag {

    /**
     * Construct the base type, Tag
     *
     * @param Array $attributes
     * @param Array $childTags
     */
    public function __construct(Array $attributes, Array $childTags)
    {
        parent::__constrct($attributes, $childTags);
    }

    /**
     * Define some output for this tag
     *
     * @param  Array  $tags
     */
    public abstract function output(Array $tags)
    {
        echo '<p>';
        PMLParser::parse($tags);
        echo '</p>';
    }
}
